CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for ui-users
 
+## [11.1.0](https://github.com/folio-org/ui-users/tree/v11.1.0) ()
+[Full Changelog](https://github.com/folio-org/ui-users/compare/v11.1.0...v11.1.0)
+* Update permissions for mod-patron. Ref UIU-3259
+
```

Comment:
```suggestion
## [11.1.0] In progress
* Update permissions for mod-patron. Ref UIU-3259

```

Reply:
In case of a breaking case, please bump up the version in package.json file line #3 -` "version": "11.0.0",`

and also update the same version on changelog too.